### ISSUE TITLE
Reaction Anim keyframe limit.

### ISSUE THREAD

**Issue by chineselectricbatman** [WolfireGames/overgrowth#129]
custom reaction anims seem to cut off even with the clock symbol ticked on so you could set the limit to 999 and the anim will only play for 2 seconds. It looks like an issue with aschar.

**Comment by kavika13** [comment]
Can you give me a small repro case? Easier to fix stuff in my spare time when I have one.

**Comment by chineselectricbatman** [comment]
I don't really know the github lingo but I'm guessing you mean show you a
reproduction of the issue?

**Comment by mallocmothra** [comment]
yes a reproduction of the issue. the animation file, and whatever additional files you can use to trigger it.

trying to cut the data for reproducing the problem down to just the minimum needed to reproduce the issue at hand would be most beneficial.

**Comment by kavika13** [comment]
yeah. providing a small set of data and steps to that are the minimum required to reproduce the problem is pretty essential in bug reports. it makes it so the devs don't have to spend time guessing at what you mean, and can dive in quickly without necessarily having previous experience with the specific functionality you're talking about.

it also means they won't accidentally do something slightly different when trying to reproduce the problem that causes the bug not to trigger.

**Comment by chineselectricbatman** [comment]
Alright I hope this is along the lines of what you're looking for.


**How to reproduce the problem**

Step 1: Following the Wolfire wiki- create your choice of animation in Blender and it's xml files to go along with them, for the issue im gonna use Throw attack as a example. In Overgrowth throw attacks have 3 different type of prerequisites. The Throw, The Thrown and the Throw counter.

Step 2: Supposing you've made your own custom attack animation longer than 40 frames, the Thrown reaction anim for your opponent will cut off without playing in its entirety ingame.

 This ain't supposed to happen as all other attack forms in the game have no limit on them really except reaction anims. If its intentional it is weirdly restrictive, its possible to fix this issue by altering aschar but I wasn't the one who patched it so I'm not really knowledgeable about how it was done. Also altering aschar brings a whole other package of issues with compatibility so its not ideal.

Hope this makes it clear what I meant, sorry for the vague nothingness report i made initially..

**Comment by kavika13** [comment]
Can you send over the animation? Either here or on the discord.

**Comment by yanwangken** [comment]
I can send the animation files and blendfile I'll do it over discord just because I don't wanna leak the blendfile in case anyone stumbles on them and nicks em.
